packages/devtools_app/lib/src/inspector/layout_explorer/ui/free_space.dart

```diff
@@ -150,10 +152,15 @@ class PaddingVisualizerWidget extends StatelessWidget {
       top: renderProperties.offset.dy,
       left: renderProperties.offset.dx,
       child: Container(
-        width: renderProperties.width,
-        height: renderProperties.height,
+        width: safeDouble(renderProperties.width),
+        height: safeDouble(renderProperties.height),
         child: horizontal ? widthWidget : heightWidget,
       ),
     );
   }
 }
+
+double safeDouble(double value) {
```

Comment:
we may already have something like this in utils.dart. if not, can you move this method there?

Reply:
done